src/test/java/core/basesyntax/DataParserTest.java

```diff
@@ -0,0 +1,68 @@
+package core.basesyntax;

```

Comment:
```suggestion
package core.basesyntax.service.impl;
```
path should be the same as in your solution